build.gradle

```diff
@@ -89,7 +89,7 @@ subprojects {
 
                 // TODO: Unreleased dependencies.
                 // These must already be installed in the local maven repository.
-                netty: 'io.netty:netty-codec-http2:4.1.0.Beta5-SNAPSHOT',
+                netty: 'io.netty:netty-codec-http2:4.1.0.Beta6-SNAPSHOT',
```

Comment:
There is actually another change that we need from beta6: https://github.com/netty/netty/commit/dca6cea78ef6f58d65125fcd268710bd86b86c22.  So I think we need to upgrade to the snapshot regardless.  Let's just cut the tag this week so we can progress on master.